src/nanovg.h

```diff
@@ -124,6 +160,20 @@ void nvgCancelFrame(NVGcontext* ctx);
 void nvgEndFrame(NVGcontext* ctx);
 
 //
+// Composite operation
+//
+// The default composite operation of NanoVG is NVG_SOURCE_OVER.
```

Comment:
I think we should mention here in the common comment that the composite operations are modeled after html canvas, and blend func is based on opengl (and see corresponding manuals for more info), and that the colors in the blending stage have premultiuplied alpha. Otherwise, we're good to go!